src/backend/gl/src/command.rs

```diff
@@ -1484,6 +1484,31 @@ impl command::CommandBuffer<Backend> for CommandBuffer {
         unimplemented!()
     }
 
+    unsafe fn draw_mesh_tasks(&mut self, _: u32, _: u32) {
```

Comment:
we have some defines for these, could we use them here, or add a new one if needed?
 I.e. look at how other draws use `hal::DrawCount`, `hal::IndexCount`, etc

Reply:
I created TaskCount and TaskOffset.